pera-execution/v1/pera-adapter/src/execution_engine.rs

```diff
@@ -671,6 +671,7 @@ mod checked {
             TransactionKind::DWalletMPCOutput(_) => {
                 panic!("DWalletMPCOutput should not exist in v1");
             }
+            TransactionKind::LockNextCommittee(..) => Ok(Mode::empty_results()),
```

Comment:
Why do you support the lock transaction in this version? And if it’s supported in v1, why not in v2?

Reply:
removed